extension.ts

```diff
@@ -67,6 +67,8 @@ const findVSCode = (): CodeAppInfo | null => {
     // Offical VSCode snap, see <https://github.com/Jomik/vscode-search-provider/pull/14>
     // and <https://snapcraft.io/code>
     ["code_code.desktop", "Code"],
+    // VSCodium support
```

Comment:
Please just add a link to this pull request here (like for the snapcraft the line above) so that we know where this comes from.  Then it's good to go :)

Reply:
OK
Done :)